go/README.md

```diff
@@ -1 +1,177 @@
-# Go
\ No newline at end of file
+# Go Code Conventions
+
+## Tooling
+
+- Visual Studio Code
+- delve
+- golangci-lint
+- staticcheck
+- modd
```

Comment:
Heeft de tooling strict te maken met de taal conventies? Mocht ik Goland helemaal fantastisch vinden en ik check geen Goland specifieke files in dan heeft het impact op de code kwaliteit toch?

Reply:
Tooling is inderdaad geen (taal)conventie; zou ik eruit halen.

Reply:
okay